programs/bench.h

```diff
@@ -121,6 +122,8 @@ BMK_return_t BMK_syntheticTest(int cLevel, double compressibility,
  * (cLevel, comprParams + adv in advanced Mode) */
 /* srcBuffer - data source, expected to be valid compressed data if in Decode Only Mode
  * srcSize - size of data in srcBuffer
+ * dstBuffer - destination buffer to write compressed output in, optional (NULL)
```

Comment:
What do you mean by (NULL). Also, it seems like this documentation is added to the wrong doc block.